packages/now-next/src/index.ts

```diff
@@ -355,6 +354,100 @@ export const build = async ({
     }
   }
 
+  const exportIntent = await getExportIntent(entryPath);
+  if (exportIntent) {
+    const { trailingSlash } = exportIntent;
+
+    const userExport = await getExportStatus(entryPath);
+    if (!userExport) {
+      await writePackageJson(entryPath, {
+        ...pkg,
+        scripts: {
+          ...pkg.scripts,
+          'now-do-not-use-export': `next export --outdir "${path.resolve(
```

Comment:
This should be renamed.
```suggestion
          'now-automatic-next-export': `next export --outdir "${path.resolve(
```